File: Programmers/코딩테스트_입문/YunJi/Day05/옷가게 할인 받기.js
function solution(price) {
  let answer = 0;
  let sale = 0;
  if (price >= 500000) {
      sale = 20
  } else if (price >= 300000) {
      sale = 10
  } else if (price >= 100000) {
      sale = 5
  }
  answer = price - ( price * sale / 100)
  // 소숫점 이하를 버린 정수 리턴
  return parseInt(answer);
}

File: Programmers/코딩테스트_입문/GangHyeok/Day05/옷가게 할인 받기.java
class Solution {
    public int solution(int price) {
        double answer = 0;
        
        if(price>=500000){
            answer = (double)price * 0.8;
        }
        else if(price >= 300000){
            answer = (double)price *0.9;
        }
        else if (price >= 100000){
            answer = (double)price *0.95;
        }
        else{
            answer = (double)price;
        }
        
        return (int)answer;
    }
}